packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { PropertyRow } from "../PropertyRow";
+import { WorkOrder } from "./types";
+import moment from "moment";
+import { Color, useToken } from "@equinor/mad-components";
+
+const defaultWorkOrderLabelMap: Record<keyof WorkOrder, string> = {
+    title: "Title",
+    workOrderId: "Work order ID",
+    maintenanceType: "Maintenance type",
+    tagId: "Tag ID",
+    equipmentId: "Equipment ID",
+    activeStatusIds: "Active status IDs",
+    basicStartDate: "Basic start",
+    basicFinishDate: "Basic finish",
+    requiredEnd: "Required end",
+    workCenterId: "Work center ID",
+    functionalLocation: "Functional Location",
+} as const;
+
+type PropertyListProps = {
+    workOrder: Partial<WorkOrder>;
+    valueColor: Color;
+    currentDate: Date;
+    overwriteLabel?: Partial<Record<keyof WorkOrder, string>>;
+};
+
+export const PropertyList = ({
+    workOrder,
+    valueColor,
+    currentDate,
+    overwriteLabel = {},
+}: PropertyListProps) => {
+    const requiredEnd = workOrder.requiredEnd ? new Date(workOrder.requiredEnd) : null;
+
+    const formatDate = (dateString: string) => moment(dateString).format("DD.MM.YYYY");
+    const token = useToken();
+
+    const getLabel = (key: keyof WorkOrder, combinedDates: string | null) => {
```

Comment:
`combinedDates` isn't necessary to have as an argument. `getLabel` has access to it